### ISSUE TITLE
0.40.0 does not seem to merge configuration correctly

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#507]
After upgrading to 0.40.0, with the following stylesheet:

```scss
* {
  box-sizing: border-box;
}
```

and an empty `.scss-lint.yml` file, I get the following warning:

> [W] Compass::PropertyWithMixin: Use the Compass `box-sizing` mixin instead of the property

If I don't have a `.scss-lint.yml` file at all, this does not happen. It seems like perhaps the configuration isn't being merged in the same way in this version.

This was not a problem under 0.39.0.

**Comment by sds** [comment]
Thanks for the report, @lencioni.

Hotfix was released in [0.40.1](https://github.com/brigade/scss-lint/releases/tag/v0.40.1).
